Automattic/wp-calypso
# [Domains] Cleanup Kracken feature flag

This change removes the `domains/kracken-ui` feature flag following the release of the Kracken domain registration redesign project. It also removes now-obsolete components, code, and styles.

# Test instructions
1. Spin this branch up locally, both in development (`npm start`) and production (`npm run build-docker && npm run docker`) environments.
2. Navigate to `/start/domains`.
3. Verify that the domain search interface works identically [as before](https://wordpress.com/start/domains).
